uint8  order_id         # The ID of the desired server action
string order_argument   # The order argument if the order supports it
---
string response         # The response from the server